config/initializers/devise_token_auth.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+DeviseTokenAuth.setup do |config|
+  # By default the authorization headers will change after each request. The
+  # client is responsible for keeping track of the changing tokens. Change
+  # this to false to prevent the Authorization header from changing after
+  # each request.
+  # config.change_headers_on_each_request = true
```

Comment:
We usually set this to false so we are able to do multiple postman queries without having to change the auth headers